### ISSUE TITLE
Updating outdated packages on Nuget.org

### ISSUE THREAD

**Issue by sgrottel** [KhronosGroup/OpenXR-SDK-Source#315]
Hello, any chance to have both open-xr nuget packages on nuget.org updated?

- https://www.nuget.org/packages/OpenXR.Loader/
- https://www.nuget.org/packages/OpenXR.Headers/

Those are quite old. The nugets are, for example, mentioned on this getting-started guide here:
https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-openxr-in-an-existing-project

It would be really nice, if the packages could be updates, especially, since the loader packages seem available in the github build artifacts anyway.

If the packages are outdated, because noone finds the time to take care of publishing them, let me know if I can help. I'd gladly work on nuspecs, scripts, pipelines/actions, etc.

**Comment by rpavlik** [comment]
Hmm, I'm looking into who uploaded those. There are nuget packages uploaded to each release here on GitHub, but I don't think I have access right now to push those to nuget.org

**Comment by sgrottel** [comment]
I just noticed, @rpavlik, since your last comment, it's now "28 Days Later" :laughing:

Any update?

**Comment by rpavlik** [comment]
Patience is appreciated. People in the wg are generally quite busy. I have pinged the appropriate person to update or take down the packages. You can get updated packages from GitHub releases.

**Comment by lewa-j** [comment]
Have you had a chance to look into this yet?

**Comment by MarkVabulas** [comment]
I would also appreciate some progress in this direction, as well.  Thanks!

**Comment by lewa-j** [comment]
Still up. Still outdated.
Also latest github release has no nupkg file

**Comment by rpavlik** [comment]
yeah, latest release isn't fully out yet, the signing step is manual. I can ping the person who put it on nuget.org in the first place again.

**Comment by lewa-j** [comment]
Please do. Thanks

**Comment by rpavlik** [comment]
current release now has nuget files, thanks for your patience.
